tests/kafkatest/tests/core/replication_test.py

```diff
@@ -138,10 +138,11 @@ def min_cluster_size(self):
     @parametrize(failure_mode="hard_bounce",
             broker_type="leader",
             security_protocol="SASL_SSL", client_sasl_mechanism="SCRAM-SHA-256", interbroker_sasl_mechanism="SCRAM-SHA-512")
-    def test_replication_with_broker_failure(self, failure_mode, security_protocol,
-                                             broker_type, client_sasl_mechanism="GSSAPI",
-                                             interbroker_sasl_mechanism="GSSAPI",
-                                             enable_idempotence=False):
+    @matrix(failure_mode=["clean_shutdown", "hard_shutdown", "clean_bounce", "hard_bounce"],
+            security_protocol=["PLAINTEXT"], broker_type=["leader"], compression_type=["gzip"])
```

Comment:
I'm OK with merging as is, if you're concerned about removing the cases I suggested.